crates/cxx-qt-lib/src/gui/qcolor.rs

```diff
@@ -9,31 +9,247 @@ use std::mem::MaybeUninit;
 
 #[cxx::bridge]
 mod ffi {
```

Comment:
need to add serde support for QColor, either with all the formats as well or converting to one of them?